docs/slack.rst

```diff
@@ -0,0 +1,36 @@
+Write the Docs Slack
+====================
+
+Our Slack channel has a bunch of interesting folks hanging out and chatting about documentation.
```

Comment:
Yea, I split things out into #meetups and #conferences -- maybe it makes sense to have one #events channel, but they seem distinct in my mind.